src/PayloadParser.h

```diff
@@ -316,9 +316,10 @@ namespace snowcrash {
             SectionType sectionType = pd.sectionContext();
 
             if (sectionType == RequestSectionType || sectionType == RequestBodySectionType) {
+
                 checkRequest(node, pd, out);
-            }
-            else if (sectionType == ResponseSectionType || sectionType == ResponseBodySectionType) {
+            } else if (sectionType == ResponseSectionType || sectionType == ResponseBodySectionType) {
```

Comment:
So, looking at the snow crash sources we have 

- `^ *\} *else if` 12-times
- `^ *else if` 52-times

is this +1 really needed? :)